eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -152,19 +152,25 @@ jobs:
         workingDirectory: $(Build.SourcesDirectory)/eng/tools/eng-package-utils
         displayName: "Get package path"
 
-      - pwsh: |
-          Start-Process "node" -ArgumentList "launch.js run start-browser-relay" -NoNewWindow -WorkingDirectory "$(Build.SourcesDirectory)/common/tools/dev-tool"
-          for ($i = 0; $i -lt 10; $i++) {
-              try {
-                  Invoke-WebRequest -Uri "http://localhost:4895/health" | Out-Null
-                  exit 0
-              } catch {
-                  Write-Warning "Failed to successfully connect to browser credential relay. Retrying..."
-                  Start-Sleep 6
-              }
-          }
-          Write-Error "Could not connect to browser credential relay."
-          exit 1
+      - task: AzurePowerShell@5
+        inputs:
+          azureSubscription: ${{ parameters.CloudConfig.ServiceConnection }}
```

Comment:
I suspect this is not setup for anyone that hasn't opted in to the new model yet. Have we tested this on any of those test pipelines to ensure this is set.

Reply:
@minhanh-phan looks like we may need to put this under the `- ${{ if eq('true', parameters.UseFederatedAuth) }}:` just below, and have the old version of the script in the corresponding else block, like how there are two versions of the other steps currently (one for the new model using Azure PowerShell and another without).

Reply:
That makes sense to me. I'll make sure to address that